Failure on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.36225&attempt_id=d6fa066d-4043-4115-99a0-6cb157ece13d)

<details><summary>Partial log (click to expand)</summary><p>

```
#include <Security/Authorization.h>
         ^~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
make[2]: *** [Makefile:388: stubby-ui-helper.o] Error 1
make[2]: Leaving directory '/private/tmp/nix-build-stubby-0.2.2.drv-0/source/macos'
make[1]: *** [Makefile:495: all-recursive] Error 1
make[1]: Leaving directory '/private/tmp/nix-build-stubby-0.2.2.drv-0/source'
make: *** [Makefile:351: all] Error 2
builder for '/nix/store/qqsx466ckbgxag8prz9jy69r2nmpaq1z-stubby-0.2.2.drv' failed with exit code 2
error: build of '/nix/store/qqsx466ckbgxag8prz9jy69r2nmpaq1z-stubby-0.2.2.drv' failed
```
</p></details>

--- a/pkgs/tools/networking/stubby/default.nix
+++ b/pkgs/tools/networking/stubby/default.nix
@@ -1,0 +1,35 @@
+{ stdenv, fetchFromGitHub, getdns, libtool, m4, file , doxygen
+, autoreconfHook, automake, check, libbsd, libyaml, darwin }:
+
+stdenv.mkDerivation rec {
+  pname = "stubby";
+  name = "${pname}-${version}";
+  version = "0.2.2";
+
+  src = fetchFromGitHub {
+    owner = "getdnsapi";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1zq7h3jx6v821phcbjp348ncdicx9s4gqkj7mcz8kd6ps902iag8";
+  };
+
+  nativeBuildInputs = [ libtool m4 libbsd libyaml autoreconfHook ];
+
+  buildInputs = [ doxygen getdns automake file check ]
+    ++ stdenv.lib.optionals stdenv.isDarwin [ darwin.Security ];
+
+  meta = with stdenv.lib; {
+    description = "A local DNS Privacy stub resolver (using DNS-over-TLS)";
+    longDescription = ''
+      Stubby is an application that acts as a local DNS Privacy stub
+      resolver (using RFC 7858, aka DNS-over-TLS). Stubby encrypts DNS
+      queries sent from a client machine (desktop or laptop) to a DNS
+      Privacy resolver increasing end user privacy. Stubby is developed by
+      the getdns team.
+'';
+    homepage = https://dnsprivacy.org/wiki/x/JYAT;
+    downloadPage = "https://github.com/getdnsapi/stubby";
+    maintainers = with maintainers; [ leenaars ];
+    license = licenses.bsd3; platforms = platforms.all;
+    };
+}

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -4781,6 +4781,8 @@ with pkgs;
 
   storebrowse = callPackage ../tools/system/storebrowse { };
 
+  stubby = callPackage ../tools/networking/stubby { };
+
   syntex = callPackage ../tools/graphics/syntex {};
 
   fusesmb = callPackage ../tools/filesystems/fusesmb { samba = samba3; };

